.clusterfuzzlite/build.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash -eu
+# Supply build instructions
+# Use the following environment variables to build the code
+# $CXX:               c++ compiler
+# $CC:                c compiler
+# CFLAGS:             compiler flags for C files
+# CXXFLAGS:           compiler flags for CPP files
+# LIB_FUZZING_ENGINE: linker flag for fuzzing harnesses
+
+# When run multiple times locally leftover files may be present in shared
+# OUT folder. Clear these in case.
+rm -f $OUT/test_file*
+
+sh autogen.sh
+./configure
+make
+make install
```

Comment:
```suggestion
make -j2 install
```
Should be a little faster.

Reply:
Done, thanks!